Extensions/3D/Cube3DRuntimeObject.ts

```diff
@@ -203,10 +206,13 @@ namespace gdjs {
       if (this._faceResourceNames[faceIndex] === resourceName) {
         return;
       }
-
       this._faceResourceNames[faceIndex] = resourceName;
       this._renderer.updateFace(faceIndex);
     }
+    setCubeColor(color: THREE.Color): void {
```

Comment:
This method is part of a public interface. We don't want to rely THREE objects for inputs. You can check `setFillColor` int he codebase to see how it's done in other objects.